docs/QUnit/module.md

```diff
@@ -268,3 +290,103 @@ QUnit.module( "Database connection", {
   }
 } );
 ```
+
+---
+
+Only run a subset of tests:
+
+```js
+QUnit.module( "Tank", function() {
```

Comment:
While I appreciate keeping with the theme of "[firin' mah lazer](https://youtu.be/YCj8Yzx15BA)", we should try to avoid "war" imagery. While likely not a big deal for some folks, guns and tanks can be sensitive subjects for others and we want to ensure the documentation we provide is welcoming to all.

Maybe we can have a module for "Android"? That does things like "program" and "actHuman"?